test/tests/python-hy/container.sh

```diff
@@ -28,8 +28,11 @@ fi
 export PIP_DISABLE_PIP_VERSION_CHECK=1
 # or that a new version will no longer work with this python version
 export PIP_NO_PYTHON_VERSION_WARNING=1
+# ensure pip does not complain about running about root
+export PIP_ROOT_USER_ACTION=ignore
+
```

Comment:
This suppresses  the following output from the test logs

```
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
```
https://doi-janky.infosiftr.net/job/update.sh/job/pypy/10376/execution/node/98/log/